drivers/postgres.go

```diff
@@ -863,32 +863,40 @@ func (db *Postgres) GetPrimaryKeyColumnNames(database, table string) (primaryKey
 
 	splitTableString := strings.Split(table, ".")
 
-	if len(splitTableString) == 1 {
+	if len(splitTableString) != 2 {
```

Comment:
This seems like a more robust way to determine of a string in the format of `schema.table` was given.